apps/src/applab/designMode.js

```diff
@@ -172,6 +176,24 @@ designMode.onPropertyChange = function(element, name, value) {
 };
 
 /**
+ * Create a data-URI with the image data of the given icon glyph.
+ * @param value
+ * @return {*|string}
```

Comment:
What does *| represent here? Might also be worth a comment on line 180 that we expect value to be in the form of icon://<icon-name>